snapcraft/cli/remote.py

```diff
@@ -150,6 +156,12 @@ def remote_build(
     _monitor_build(lp)
 
 
+def _clean_build(lp: LaunchpadClient):
+    echo.info("Cleaning existing builds and artifacts...")
```

Comment:
s/builds/tasks/ or jobs

Reply:
Pulled out and simplified to "Cleaning..." https://github.com/snapcore/snapcraft/pull/2775